src/components/problem/content.vue

```diff
@@ -18,7 +20,7 @@
       </div>
       <div id="pannel" v-if="!isWider">
         <div id="tools">
-          <el-card>
+          <el-card v-if="showSmallToolBar || !smallScreen">
```

Comment:
You are getting this file more and more complex... That's not good.
Well... I'll teach you how to refactory this later we back to school.

Reply:
The real COMPLEX comes after I tell you the huge and UNFIXABLE bug of list tag, wait for it